seatunnel-e2e/seatunnel-connector-v2-e2e/connector-redis-e2e/pom.xml

```diff
@@ -18,30 +18,32 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
         <groupId>org.apache.seatunnel</groupId>
-        <artifactId>seatunnel-flink-connector-v2-e2e</artifactId>
+        <artifactId>seatunnel-connector-v2-e2e</artifactId>
         <version>${revision}</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>connector-redis-flink-e2e</artifactId>
+    <artifactId>connector-redis-e2e</artifactId>
 
     <dependencies>
         <dependency>
-            <groupId>org.apache.seatunnel</groupId>
-            <artifactId>connector-flink-e2e-base</artifactId>
-            <version>${project.version}</version>
-            <classifier>tests</classifier>
-            <type>test-jar</type>
+            <groupId>redis.clients</groupId>
+            <artifactId>jedis</artifactId>
```

Comment:
remove?
already exists in `connector-redis`

Reply:
My fault, dependencies that are not provided don't need to be handled like this.